server/src/models/user.js

```diff
@@ -0,0 +1,79 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+const userSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+        trim: true,
+        min: 3,
+        max: 20
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true,
+        min: 3,
+        max: 20
+    },
```

Comment:
There is no need for max 20 characters on names

Reply:
It depends, nothing wrong with it

Reply:
@himanshujaidka I know it depends, but it's not like there doesn't exists a name with more than 20 chars. That's not definite so it's not recommended to restrict this